packages/io/__tests__/router.ts

```diff
@@ -0,0 +1,35 @@
+import { describe, it, expect } from '@jest/globals'
+import { CoR } from '../src/cor'
+import { KuaiRouter } from '../src/router'
+
+describe('test KuaiRouter', () => {
+  it(`should use kuaiRouter`, async () => {
+    const cor = new CoR()
+    const kuaiRouter = new KuaiRouter()
+    kuaiRouter.get('/', async (ctx, next) => {
+      ctx.ok('hello root')
+      await next()
```

Comment:
`ctx.ok('hello root')` resolves the promise returned by `cor.dispatch` method, what does the following `next` do here

Reply:
next() really doesn't do anything, it has been removed